sasmodels/data.py

```diff
@@ -129,11 +129,6 @@ def set_top(data, cutoff):
         Boxcut(x_min=-np.inf, x_max=np.inf, y_min=-np.inf, y_max=cutoff)(data)
 
 
-class Source:
-    ...
-class Sample:
-    ...
-
```

Comment:
These are used.

Reply:
`Source` and `Sample` are defined twice within the file. Once here, on lines 132-135, and then on lines 311-326, where the definition is more than an ellipsis. Let me know whether the other definition should be removed instead.

Reply:
okay, that's a bug